login/login.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobby</title>
+    <link rel="stylesheet" 
+		href="https://kaizen-ui.netlify.app//styles.css">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" 
+	  href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <nav class="navbar_std">
+        <header class="d-flex justify-space-bw align-center">
+            <div class="brandName"><a href="../index.html">Mobby</a></div>
+            <div class="searchBar"><i class="fa fa-search"></i><input type="search" placeholder="search"></div>
+            <div id="hambar" class="hambar"><i class="fa fa-bars" aria-hidden="true"></i></div>
+            <ul class="navbarOptions d-flex d-flex justify-space-bw align-center">
```

Comment:
you should follow same convention for naming css classes. here you have used camelCase and `-` both

Reply:
In components I have followed camelcase. That why I am following for better understanding the code.